src/lib/components/product.svelte

```diff
@@ -2,27 +2,21 @@
   import { fade } from 'svelte/transition'
 
   export let index: number
-  export let image: string
-  export let name: string
+  export let cardImage: string
```

Comment:
why is this better?

Reply:
because there are two sizes of images, the one on the card and the one on the product description, use cardImage to differentiate from the other image

Reply:
this component still only has one image prop